napari/components/_viewer/view/main.py

```diff
@@ -71,6 +74,8 @@ def __init__(self, viewer):
                 'standard': QCursor()
             }
 
+        self._generators = {}
```

Comment:
can this have a more specific name like `key_generators` or something else

Reply:
I've renamed it to `_key_release_generators`